project/src/main/webapp/createUserAccount.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Cloudberry</title>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <link rel="stylesheet" href="https://code.getmdl.io/1.3.0/material.teal-orange.min.css" />
+    <script defer src="https://code.getmdl.io/1.3.0/material.min.js"></script>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <!-- Always shows a header, even in smaller screens. -->
+    <div class="mdl-layout mdl-js-layout mdl-layout--fixed-header">
+      <header class="mdl-layout__header">
+        <div class="mdl-layout__header-row">
+          <!-- Title -->
+          <span class="mdl-layout-title">CloudBerry</span>
+          <!-- Add spacer, to align navigation to the right -->
+          <div class="mdl-layout-spacer"></div>
+          <!-- Navigation. We hide it in small screens. -->
+          <nav class="mdl-navigation mdl-layout--large-screen-only">
+            <a class="mdl-navigation__link" href="">Browse</a>
+            <a class="mdl-navigation__link" href="">About</a>
+            <a class="mdl-navigation__link" href="">Contact</a>
+            <a class="mdl-navigation__link" href="">Account</a>
+          </nav>
+        </div>
+      </header>
+      <div class="mdl-layout__drawer">
+        <span class="mdl-layout-title">CloudBerry</span>
+        <nav class="mdl-navigation">
+          <a class="mdl-navigation__link" href="">Browse</a>
+          <a class="mdl-navigation__link" href="">About</a>
+          <a class="mdl-navigation__link" href="">Contact</a>
+          <a class="mdl-navigation__link" href="">Account</a>
+        </nav>
+      </div>
+      <main class="mdl-layout__content">
+        <div class="page-content" id="content">
+          <!-- Your content goes here -->
+          <h1>Create a User Account</h1>
+          <!-- TODO: Have a login and sign up servlet -->
+          <form action="" method="POST" >
+            <p>Username*</p> 
+            <div class="mdl-textfield mdl-js-textfield">
+              <input class="mdl-textfield__input" type="text" id="nickname" name="nickname" required>
+              <label class="mdl-textfield__label" for="nickname">JohnDoe42</label>
+            </div><br/><br/>
+            <p>Address*</p>
```

Comment:
You're collecting this so you can show the localized search results, right? It might be nice to at some point add a note about what it will be used for, so the user will know why you want to know.